app/views/capture/_save.html.erb

```diff
@@ -17,7 +17,7 @@
 
 <form class="form" action="/spectrums" method="post" id="saveForm" target="_blank" onSubmit="$W.cancelSave();window.location='#page1'">
 
-  <input placeholder="Title (required)" name="spectrum[title]" id="spectrum_notes" type="text" /><br />
+  <input placeholder="Title (required)" name="spectrum[title]" type="text" /><br />
```

Comment:
Oops, are you sure here? This is the legacy capture interface, would this break it?

Reply:
A warning was popping up in the console that there were 2 id's associated with the same name - `spectrum_notes`: 

https://github.com/publiclab/spectral-workbench/blob/8f89e0fffd0ac02068857ed6fb775aa5a4460591/app/views/capture/_save.html.erb#L20

https://github.com/publiclab/spectral-workbench/blob/8f89e0fffd0ac02068857ed6fb775aa5a4460591/app/views/capture/_save.html.erb#L55

I don't the id being used anywhere, so maybe it is ok to proceed 🤞

Reply:
hm, ok! As long as you can confirm it's not broken, sounds good! And indeed, a quick search shows it's not used in code anywhere -- https://github.com/publiclab/spectral-workbench/search?q=spectrum_notes

looks like it was simply duplicated! Good catch!